Makefile

```diff
@@ -210,6 +211,8 @@ config.h:
 	echo '/* Default config.h generated by Makefile */' > $@
 	echo '#define HAVE_LIBBZ2 1' >> $@
 	echo '#define HAVE_LIBLZMA 1' >> $@
+	echo '#define HAVE_FSEEKO 1' >> $@
+	echo '#define HAVE_DRAND48 1' >> $@
```

Comment:
The only place that needs to use `fseeko()` is `zfseeko()`, which is itself unused.  Moreover for _zfio.c_ in general: if someone is in a tidying mood, see #552.